docs/source/ufs_utils.rst

```diff
@@ -544,14 +544,22 @@ The surface climatological data is located here `./fix/fix_sfc_climo <https://no
              * N Hemis 30 sec - soil_type.statsgo.nh.30s.nc
              * Global 30 sec - soil_type.statsgo.30s.nc
       * Categorical BNU soil type
```

Comment:
Remove merge conflicts.

Reply:
I removed a conflict in sfc climo.  I see these merge conflicts in many files. Removing them now